docs/api/model_selection/cross_validation.rst

```diff
@@ -70,13 +72,16 @@ In this next example we show how to visualize cross-validated scores for a regre
     from sklearn.model_selection import KFold
 
 
-    energy = load_data("energy")
+    # Load the regression data set
+    data = load_data("energy")
 
+    # Specify the features of interest and the target
     targets = ["heating load", "cooling load"]
-    features = [col for col in energy.columns if col not in targets]
+    features = [col for col in data.columns if col not in targets]
```

Comment:
Since I renamed `energy` to `data` to load the dataset, I needed to make that change again here.